Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2702,7 +2692,9 @@ - (void)messageWasSent:(TSOutgoingMessage *)message
     self.inputToolbar.quotedReply = nil;
 
     if ([Environment.preferences soundInForeground]) {
-        [JSQSystemSoundPlayer jsq_playMessageSentSound];
+        SystemSoundID soundId = [OWSSounds systemSoundIDForSound:OWSSound_MessageSent quiet:YES];
+        // Vibrate, respect silent switch, respect "Alert" volume, not media volume.
```

Comment:
Do we want to vibrate, respect silent switch, and respect alert volume here?

Can you compare with behavior in messages/whatsapp - I'd be surprised.

Reply:
You're right, we should be using `AudioServicesPlaySystemSound()` not `AudioServicesPlayAlertSound()` as I had it to match previous behavior in `JSQSystemSoundPlayer jsq_playMessageSentSound`